src/main/java/io/confluent/connect/jdbc/sink/dialect/PostgreSqlDialect.java

```diff
@@ -97,4 +97,18 @@ public void apply(StringBuilder builder, String col) {
     return builder.toString();
   }
 
+  @Override
+  protected void formatColumnValue(StringBuilder builder, String schemaName, Map<String, String> schemaParameters, Schema.Type type, Object value) {
+    if (schemaName != null) {
+      super.formatColumnValue(builder, schemaName, schemaParameters, type, value);
+    }
```

Comment:
Good catch. I need to think about the best way to solve this. This method just mutates `builder` which makes it harder to override in the subclass. There's no default case in the superclass, so `return` is only valid if it matches one of the cases in the superclass, but it's not clear what the best way to determine whether the superclass handled the particular schema name or not is.